src/bundle/Resources/public/js/scripts/fieldType/ezimage.js

```diff
@@ -113,7 +114,7 @@
                     eventName: 'blur',
                     callback: 'validateAltInput',
                     invalidStateSelectors: ['.ibexa-data-source__field--alternativeText'],
-                    errorNodeSelectors: [`${SELECTOR_ALT_WRAPPER} .ibexa-data-source__label-wrapper`],
```

Comment:
Question: isn't this BC?

Reply:
I don't think so, especially as this selector is incorrect, as errors shouldn't be displayed in label block